dist/textbox/textbox.css

```diff
@@ -120,6 +122,11 @@ textarea.textbox__control:not(:read-only):focus {
   background-color: var(--textbox-focus-background-color, var(--color-background-primary));
   outline: 0;
 }
+input.textbox__control:read-only:focus,
+textarea.textbox__control:read-only:focus {
+  outline: 0;
+  text-decoration: underline;
```

Comment:
Is this a new design standard?

Reply:
This is what we have for borderless inputs like select. Because otherwise the focus rings are almost overlapping the text

Reply:
<img width="659" alt="Screen Shot 2022-07-27 at 3 28 16 PM" src="https://user-images.githubusercontent.com/1755269/181383448-3eac6878-2865-4a39-95f1-253624b68f23.png">
FYI, this is what it looks like with focus rings without padding